evtgen.sh

```
@@ -35,22 +37,8 @@ cmake $SOURCEDIR -DCMAKE_INSTALL_PREFIX=$INSTALLROOT \
-  puts stderr "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
-}
-set version $PKGVERSION-@@PKGREVISION@$PKGHASH@@
-module-whatis "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
-# Dependencies
-module load BASE/1.0 pythia/$PYTHIA_VERSION-$PYTHIA_REVISION ${HEPMC3_VERSION:+HepMC3/$HEPMC3_VERSION-$HEPMC3_REVISION} ${TAUOLAPP_VERSION:+Tauolapp/$TAUOLAPP_VERSION-$TAUOLAPP_REVISION} ${PHOTOSPP_VERSION:+PHOTOSPP/$PHOTOSPP_VERSION-$PHOTOSPP_REVISION}
-# Our environment
-setenv EVTGEN_ROOT \$::env(BASEDIR)/$PKGNAME/\$version
-setenv EVTGENDATA \$::env(EVTGEN_ROOT)/share/EvtGen
-prepend-path LD_LIBRARY_PATH \$::env(EVTGEN_ROOT)/lib
-$([[ ${ARCHITECTURE:0:3} == osx ]] && echo "prepend-path DYLD_LIBRARY_PATH \$::env(EVTGEN_ROOT)/lib")
+mkdir -p "$INSTALLROOT/etc/modulefiles"
```

Shellcheck (0.11.0)

[info] Command appears to be unreachable. Check usage (or ignore if invoked indirectly).

(SC2317)